Gemfile

```diff
@@ -1,3 +1,3 @@
 source 'https://rubygems.org/'
-gem 'spree', github: 'spree/spree', branch: '2-4-stable'
```

Comment:
Probably should refer to solidus here. Ideally [in the way we do elsewhere](https://github.com/solidusio/solidus_auth_devise/blob/master/Gemfile#L3-L4)